### PR TITLE
Improve the hyper-schema abstract

### PR DESCRIPTION
**Pull request by handrews**
Use language more in line with recent changes and recently
introduced concepts.  Add a little more detail about the
general capabilities.  This is all roughly inspired by reading
the guidelineson abstracts in RFC 7322 section 4.3

### CODE REVIEW

File: jsonschema-hyperschema.xml [hunk 1]
```diff
@@ -57,10 +57,13 @@
 
         <abstract>
             <t>
-                JSON Schema is a JSON based format for describing JSON data using various
-                vocabularies.  This document specifies a vocabulary
-                for annotating JSON documents with hyperlinks and instructions for processing and
-                manipulating remote JSON resources through hypermedia environments such as HTTP.
+                JSON Schema is a JSON based-format for describing JSON data using various
```
Comment (awwright) [inline jsonschema-hyperschema.xml]: Do you mean "JSON-based format"?

Comment (handrews) [reply jsonschema-hyperschema.xml]: Heh.  Yup, thanks, will fix.

**Comment by handrews** [comment]
Given the two approvals and the only comment from the third person being a trivial typo, I'm going to go ahead and merge now the the typo is fixed.

**Comment by handrews** [comment]
@awwright good point on "vocabularies".

**Pull request merged**